src/api/routes/author.js

```diff
@@ -58,21 +196,97 @@ router.get('/:bbid',
 	});
 
 
+/**
+ *	@swagger
+ * '/author/{bbid}/aliases':
+ *   get:
+ *     tags:
+ *       - Lookup Requests
+ *     summary: Get list of aliases of an author by BBID
+ *     description: Returns the list of aliases of an author
+ *     operationId: getAliasesOfAuthorByBbid
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: bbid
+ *         in: path
+ *         description: BBID of author to return that data
```

Comment:
"BBID of the author"

Reply:
I meant only that:
"description: BBID of the Author". The "to return that data" seems unnecessary to me.